Classes/Controller/LogsController.php

```diff
@@ -56,7 +56,7 @@ class LogsController extends AbstractModuleController
     /**
      * Renders the app to interact with the nodetype graph
      */
-    public function indexAction(): void
+    public function indexAction(int $limit = 10): void
```

Comment:
How about we introduce an actual pagination here. So like
```suggestion
    public function indexAction(int $page = 0): void
```

And read the `limit` from the Settings yaml?